updatecli/updatecli.d/debian.yaml

```
@@ -0,0 +1,76 @@
+---
+name: Bump Debian Bookworm version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  bookwormLatestVersion:
+    kind: dockerimage
+    name: "Get the latest Debian Bookworm Linux version"
+    spec:
+      image: "debian"
+      tagFilter: "bookworm-*"
+      versionfilter:
+        kind: regex
+        pattern: >-
+          bookworm-\d+$
+
+conditions:
+  testDockerfileArg:
+    name: "Does the Dockerfile have an ARG instruction for the Debian Bookworm Linux version?"
+    kind: dockerfile
+    disablesourceinput: true
+    spec:
+      file: debian/Dockerfile
+      instruction:
+        keyword: "ARG"
+        matcher: "DEBIAN_RELEASE"
+  testVersionInBakeFile:
+    name: "Does the bake file have variable DEBIAN_RELEASE?"
+    kind: file
+    disablesourceinput: true
+    spec:
+      file: docker-bake.hcl
+      matchpattern: "(.*DEBIAN_RELEASE.*)"
+
+targets:
+  updateDockerfile:
+    name: "Update the value of the base image (ARG DEBIAN_RELEASE) in the Dockerfile"
+    kind: dockerfile
+    sourceid: bookwormLatestVersion
+    spec:
+      file: debian/Dockerfile
+      instruction:
+        keyword: "ARG"
+        matcher: "DEBIAN_RELEASE"
+    scmid: default
+  updateDockerBake:
+    name: "Update the value of the base image (ARG DEBIAN_RELEASE) in the docker-bake.hcl"
+    kind: file
+    sourceid: bookwormLatestVersion
+    spec:
+      file: docker-bake.hcl
+      matchpattern: >-
+        variable(.*)"DEBIAN_RELEASE"(.*){(.*)(\r\n|\r|\n)(.*)default(.*)=(.*)
+      replacepattern: >-
+        variable${1}"DEBIAN_RELEASE"${2}{${3}${4}${5}default${6}= "{{ source "bookwormLatestVersion" }}"
+    scmid: default
+actions:
```

```suggestion

actions:
```
(nit, spacing between sections like above)